microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -1245,7 +1284,23 @@
             }
         );
 
+        // First, hide all of the site-specific content
+        const all_site_specific_elements = document.getElementsByClassName("site_specific_content");
+        for (let i = 0; i < all_site_specific_elements.length; i++) {
+            all_site_specific_elements[i].style.display = "none";
+        }
+
+        // Then, display the site-specific content for our current dataset
+        const site_specific_elements = document.getElementsByClassName("site_specific_content-" + state.dataset_site.value);
+        for (let i = 0; i < site_specific_elements.length; i++) {
+            site_specific_elements[i].style.display = "";
+        }
 
+        // Hide Microbiome Maps for skin
```

Comment:
We don't have any pcoa plots for skin, so we need to hide the links to that tab.